client/me/security-2fa-backup-codes-list/index.jsx

```diff
@@ -142,7 +142,7 @@ class Security2faBackupCodesList extends React.Component {
 	};
 
 	getBackupCodeHTML = codes => {
-		const datePrinted = this.props.moment().format( 'MMM DD, YYYY @ h:mm a' );
+		const datePrinted = this.props.moment().format( 'LL @ LT' );
```

Comment:
Should it be `ll @ LT` instead, because `MMM` would display only the first three letters of the month?